eng/pipelines/checkout-unix-task.yml

```diff
@@ -1,11 +1,5 @@
 # Shallow checkout sources on Unix
 steps:
-  - checkout: none
-
-  - script: |
-      set -x
-      git init
-      git remote add origin "$(Build.Repository.Uri)"
-      git fetch --progress --no-tags --depth=1 origin "$(Build.SourceVersion)"
-      git checkout "$(Build.SourceVersion)"
-    displayName: Shallow Checkout
+  - checkout: self
```

Comment:
I don't think 2 files are needed for this anymore given that both do the same thing now. Not sure if you want to make that many changes though

Reply:
I am happy to clean this up and backport it.